src/components/Todos.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import TodoItem from './TodoItem';
+
+export default class Todos extends Component {
+
+    render() {
+        console.log(this.props.todos)
+        return this.props.todos.map((todo) => (
```

Comment:
use error handling condition
like:- this.props&&this.props.todos&&this.props.todos.map((todo) => (